src/components/events/EventPosterUpload.tsx

```
@@ -34,12 +41,35 @@ export const EventPosterUpload: React.FC<EventPosterUploadProps> = ({ eventId, o
+            return;
+        }
+
+        setViolations([]);
+        setPendingFile(null);
+        const success = await uploadAndProcess(eventId, file, altText);
         if (success) {
             onUploadComplete();
         }
     };
 
+    const handleBypass = async () => {
```

ESLint

[error] Delete `··`

(prettier/prettier)